translations/ru_RU.ts

```diff
@@ -1,966 +1,973 @@
     </message>
     <message>
         <source>Ctrl+Alt+W</source>
-        <translation type="unfinished"></translation>
+        <translation>Ctrl+Alt+W</translation>
     </message>
     <message>
         <source>Close All</source>
-        <translation type="unfinished"></translation>
+        <translation>Закрыть все</translation>
     </message>
     <message>
         <source>Close All the tabs</source>
-        <translation type="unfinished"></translation>
+        <translation>Закрыть все вкладки</translation>
     </message>
     <message>
         <source>Ctrl+Shift+W</source>
-        <translation type="unfinished"></translation>
+        <translation>Ctrl+Shift+W</translation>
     </message>
     <message>
         <source>Quit</source>
-        <translation type="unfinished"></translation>
+        <translation>Выйти</translation>
     </message>
     <message>
         <source>Quit the application</source>
-        <translation type="unfinished"></translation>
+        <translation>Выйти из программы</translation>
     </message>
     <message>
         <source>Ctrl+Q</source>
-        <translation type="unfinished"></translation>
+        <translation>Ctrl+Q</translation>
     </message>
     <message>
         <source>Open File...</source>
-        <translation type="unfinished"></translation>
+        <translation>Открыть файл....</translation>
     </message>
     <message>
         <source>Open files</source>
-        <translation type="unfinished"></translation>
+        <translation>Открыть файлы</translation>
     </message>
     <message>
         <source>Ctrl+O</source>
-        <translation type="unfinished"></translation>
+        <translation>Ctrl+O</translation>
     </message>
     <message>
         <source>Open Contest...</source>
-        <translation type="unfinished"></translation>
+        <translation>Открыть соревнование...</translation>
     </message>
     <message>
         <source>Open a Contest</source>
-        <translation type="unfinished"></translation>
+        <translation>Открыть соревнование</translation>
     </message>
     <message>
         <source>Ctrl+Shift+O</source>
-        <translation type="unfinished"></translation>
+        <translation>Ctrl+Shift+O</translation>
     </message>
     <message>
         <source>Indent</source>
-        <translation type="unfinished"></translation>
+        <translation>Отступ</translation>
     </message>
     <message>
         <source>Ctrl+]</source>
-        <translation type="unfinished"></translation>
+        <translation>Ctrl+] (Ctrl+Ъ)</translation>
     </message>
     <message>
         <source>Unindent</source>
-        <translation type="unfinished"></translation>
+        <translation>Убрать отступ</translation>
     </message>
     <message>
         <source>Ctrl+[</source>
-        <translation type="unfinished"></translation>
+        <translation>Ctrl+[ (Ctrl+Х)</translation>
```

Comment:
![image](https://user-images.githubusercontent.com/30581822/87490163-94518600-c677-11ea-9076-bafd4a129eb2.png)

They are not working.

Reply:
![image](https://user-images.githubusercontent.com/32099652/87557533-14540c00-c6c9-11ea-896b-e2e01df8f6a8.png)
Is it better?

Reply:
> Is that better now?

You have to open a new window or drag the splitter to make the right part smaller so that there's a scroll bar. Otherwise, the old sizes will be loaded and the expected part is still bigger.